test/connector.ts

```diff
@@ -237,124 +240,68 @@ t.test('start only a single instance info per connection name', async t => {
   });
 });
 
-t.test('Connector reusing instance on mismatching auth type', async t => {
-  setupCredentials(t);
-
-  // mocks sql admin fetcher and generateKeys modules
-  // so that they can return a deterministic result
-  const {Connector} = t.mockRequire('../src/connector', {
-    '../src/sqladmin-fetcher': {
-      SQLAdminFetcher: class {
-        getInstanceMetadata() {
-          return Promise.resolve({
-            ipAddresses: {
-              public: '127.0.0.1',
-            },
-            serverCaCert: {
-              cert: CA_CERT,
+t.test(
```

Comment:
When we changed the changes the cache behavior so that it caches using the cache key 
{ domain/instance, ipType, iamEnabled }, the test changed too.